scripts/travis/build.sh

```diff
@@ -12,6 +12,7 @@ if [ $JOB = "unit" ]; then
     BROWSERS="SL_Chrome,SL_Safari,SL_Firefox,SL_IE_9,SL_IE_10,SL_IE_11,SL_iOS"
   fi
 
+  grunt ddescribe-iit
```

Comment:
TBH, I would prefer running all ci-checks (except for `jscs`) first and just run `jscs` after the unit tests.